src/Core/Random/UniformSphereSampler.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <Core/Random/SphereSampler.hpp>
+
+namespace Ra {
+namespace Core {
+namespace Random {
+
+class RA_CORE_API UniformSphereSampler : public SphereSampler<UniformSphereSampler>
+{
+  public:
+    UniformSphereSampler() {};
+    ~UniformSphereSampler() {};
+
+    static std::pair<Vector2, Scalar> getPointImplem( UniformGenerator* generator );
```

Comment:
```suggestion
    static std::pair<Vector2, Scalar> getPoint( UniformGenerator* generator );
```
What is the meaning of Point ? State this a a short doc.